examples/cdi-produces-disposes/README_pt.adoc

```diff
@@ -0,0 +1,291 @@
+= CDI Produces Disposes
+:index-group: CDI
+:jbake-type: page
+:jbake-status: published
+
+Esté exemplo demostra como as anotaçõẽs @Produces e @Disposes trabalham. Um LogFactory cria uma instância do LogHandler
+dependendo de um atributo "type". Para o propósito deste exemplo, o type é hard-coded para especificar um valor.
+Uma implementação Logger deve contér uma lista de LogHandlers. Teremos três implementações da interface LogHandler.
+
+* Uma DatabaseHandler
+* Uma FileHandler
+* Uma ConsoleHandler
+
+O DatabaseHandler aparentemente gravar os logs em um banco de dados. O FileHandler aparentemente gravar os logs em um arquivo.
+O ConsoleHandler seria apenas imprimir os logs para um console (saída padrão). Esté exemplo é apenas uma ilustração de como
```

Comment:
"seria apenas imprimir"

Reply:
What is wrong here?

Reply:
Same as the previous answer. O autor usa novamente "would", dando nesse caso o sentido na frase de "O FileHandler gravaria os logs em um arquivo.".

Reply:
done.